foreman/foreman.spec

```diff
@@ -983,19 +1014,31 @@ exit 0
 %{foreman_rake} db:seed >> %{_localstatedir}/log/%{name}/db_seed.log 2>&1 || :
 %{foreman_rake} apipie:cache:index >> %{_localstatedir}/log/%{name}/apipie_cache.log 2>&1 || :
 (/sbin/service foreman status && /sbin/service foreman restart) >/dev/null 2>&1
+(/sbin/service %{tasks_service_name} status && /sbin/service %{tasks_service_name} restart) >/dev/null 2>&1
 exit 0
 
 %preun
 if [ $1 -eq 0 ] ; then
 /sbin/service %{name} stop >/dev/null 2>&1
 /sbin/chkconfig --del %{name} || :
 fi
+%if 0%{?rhel} == 6
```

Comment:
Technically there is no support for EL6 anymore. Which, the `BuildRequires: systemd` would break that anyway.